### PR TITLE
enable linux/mac wasm backend test on travis-ci

### PR DESCRIPTION
**Pull request by zhaoming0**
enable Linux/Mac WASM backend test case on travis-ci

### CODE REVIEW

**Comment by zhaoming0** [comment]
@BruceDai PTAL

File: travis-ci.js [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+var csvPass = null;
+var csvFail = null;
+var csvNA = null;
+var csvExecution = "auto";
+var csvSuite = "tests";
+
+(async function() {
+    let baselinejson = JSON.parse(fs.readFileSync("./test/tools/CI/baseline/baseline.config.json"));
+    let sys = os.type();
+    let platform
```
Comment (ibelem) [inline travis-ci.js]: Please keep semicolon for this file if you are using semicolons after every statement.

File: travis-ci.js [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+            "Windows-WebGL2",
+            "Linux-clDNN",
+            "Linux-WASM",
+            "Linux-WebGL2"
+        ];
+        let backends = [
+            "WASM"
+            // "WebGL2"
+        ]
+        await driver.get('chrome://gpu');
```
Comment (ibelem) [inline travis-ci.js]: Please keep consistent double quotation marks and single quotation marks usage.

File: travis-ci.js [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+var csvModule = null;
+var csvName = null;
+var csvPass = null;
+var csvFail = null;
+var csvNA = null;
+var csvExecution = "auto";
+var csvSuite = "tests";
+
+(async function() {
+    let baselinejson = JSON.parse(fs.readFileSync("./test/tools/CI/baseline/baseline.config.json"));
```
Comment (ibelem) [inline travis-ci.js]: indent format:
```
  .html, package.json, .yml: two blanks
  baseline.config.json, travis-ci.js: Tab
```
Shall we keep two blanks for baseline.config.json, travis-ci.js?

**Comment by ibelem** [comment]
LGTM, thanks!

**Pull request merged**